lib/geordi/commands/setup.rb

```diff
@@ -28,7 +28,10 @@ def setup
   Interaction.success 'Successfully set up the project.'
 
   Hint.did_you_know [
-    :update
+    :update,
+    :security_update,
+    [:setup, :dump],
+    [:setup, :test]
```

Comment:
It is a good practice to always add a trailing comma. This way, appending another entry won't require to touch other lines (as happened in L31).